tests/other/metadce/hello_world_O3_MAIN_MODULE_2.exports

```diff
@@ -2,6 +2,9 @@ __assign_got_enties
 __errno_location
 __stdio_write
 __wasm_call_ctors
+dynCall_ii
+dynCall_iiii
+dynCall_jiji
 main
```

Comment:
why did these and the other tests change?

Reply:
I think this is because  the change to mergeLibSymbols.   Previously we were ignoring any dynCalls exported by side modules (we were only considering symbols starting with "_").   Having said that I'm not sure exactly why this file changed.. since its looks like it related purely to the main module.    Do you think want me to come up with more precise answer?

Reply:
It's possible that seeing `Module['dynCall_' + sig]` in the JS is enough for metadce to give up on optimizing away unused dynCalls, and preserve them all. A quick test could be to remove that one line to see how that changes the output.

It does seem like the right behavior to keep those functions, though! So if that is the reason, this may fix another issue I think.